### DISCUSSION TITLE
ETA for Franken UI v2?

### DISCUSSION THREAD

**Discussion by tonidy** [franken-ui/ui#46] [General]
Do you have an ETA for when Franken UI v2 will be released?

**Comment by sveltecult** [comment]
Plan was March to May 2025. It may change, though.

**Comment by tonidy** [comment]
@sveltecult thanks for the update! So, Franken UI version 2 shouldn’t be used in production yet? Hopefully, the go-live release will be ready before March 2025 🤞
